test/build/60_force.bats

```diff
@@ -61,10 +42,44 @@ RUN ["apt-get", "install", "-y", "hello"]
 EOF
     echo "$output"
     [[ $status -eq 0 ]]
-    [[ $(echo "$output" | grep -Fc 'init step 1: checking: $') -eq 1 ]]
-    [[ $(echo "$output" | grep -Fc 'init step 1: $') -eq 1 ]]
     [[ $(echo "$output" | grep -Fc 'RUN: new command:') -eq 2 ]]
-    [[ $output = *'init: already initialized'* ]]
-    [[ $output = *'--force: init OK & modified 2 RUN instructions'* ]]
+    [[ $output = *'--force=seccomp: modified 2 RUN instructions'* ]]
     [[ $output = *'grown in 4 instructions: tmpimg'* ]]
 }
+
+@test "${tag}: dpkg(8)" {
```

Comment:
Why don't these test have any `status -eq 0` or checks output?

Reply:
Any command not wrapped in `run` will fail the test if it fails (i.e., `[[ $status -eq 0 ]]` is implied). So here, we only care about whether the build succeeds or fails, not any output. Hence no need for `run`.